src/crimson/osd/main.cc

```diff
@@ -242,7 +243,9 @@ int main(int argc, char* argv[])
             seastar::engine().exit(1);
           }).get();
         }
-        fetch_config().get();
+        if (config.count("no-mon-config") == 0) {
+          fetch_config().get();
+	}
```

Comment:
@rzarzynski could you use spaces for indent?

Reply:
sure, will fix.

Reply:
Fixed.